massa-protocol-worker/src/protocol_worker.rs

```diff
@@ -292,9 +296,14 @@ impl ProtocolWorker {
                     self.update_ask_block(&mut block_ask_timer).await?;
                 }
 
-                // operation ask timer
+                // operation ask, and announce, timer
                 _ = &mut operation_batch_proc_period_timer => {
```

Comment:
It should be a different timer

Reply:
Why? It seems to make sense to announce, and ask, at the same interval.

Reply:
Ah ok I see that the ask is not done at a fixed interval.

Reply:
The operation asking system and the announcement batching system are separate, independent mechanisms. Let's keep the configs separate for more flexibility